test/opt/fold_spec_const_op_composite_test.cpp

```diff
@@ -851,7 +941,7 @@ INSTANTIATE_TEST_SUITE_P(
               {
                 "%int_n1 = OpConstant %int -1",
```

Comment:
Since this is no longer used, it should be removed.

Reply:
This is expected code, and it does appear in the output.